templates/Makefile.erb

```diff
@@ -79,8 +79,11 @@ cleanup-components: <%= @components.map {|comp| "#{comp.name}-cleanup" }.join("
 	touch <%= comp.name %>-patch
 
 <%= comp.name %>-configure: <%= comp.name %>-patch <%= list_component_dependencies(comp).join(" ") %>
+	<%- if comp.get_build_dir -%>
+	[ -d <%= comp.get_build_dir %> ] || mkdir <%= comp.get_build_dir %>
```

Comment:
Ahhh, I was about to comment - I ran this on Arch Linux and EL6 and it had this behavior. Weird about `mkdir -p`. But yeah - adding the check makes life easier for all involved.